Correct offsets for the iPhone 5,3 9.3.2

EGYbkgo9449 opened the pull request:
Correct offsets for the iPhone 5,3 9.3.2

i0mar committed:
Correct offsets for the iPhone 5,3 9.3.2

Correct offsets for the iPhone 5,3 9.3.2

benjamin-42 commented on Trident/offsetfinder.c:
```diff
@@ -16,1104 +16,1161 @@
-    determineTarget("iPhone5,3", "9.3.3", iPhone53_iOS933);
-    determineTarget("iPhone5,4", "9.3.2", iPhone54_iOS932);
-    determineTarget("iPad2,1", "9.1", iPad21_iOS910);
+	// iPhone 4S (iPhone4,1)
```
@EGYbkgo9449 could you please only commit specific changes for iPhone5,3 iOS 9.3.2? And also check the indentation, should be 4 spaces

EGYbkgo9449 replied:
Done, sry for that

i0mar committed:
Correct iPhone 5,3 9.3.2 offsets

Correct iPhone 5,3 9.3.2 offsets,
It fixes the first issue but there's another issue that happens at patching the task_for_pid

i0mar committed:
Update offsetfinder.c

i0mar committed:
Update offsetfinder.c

benjamin-42 commented on Trident/offsetfinder.c:
```diff
@@ -1,4 +1,10 @@
-//
+uint32_t find_OSSymbol_getMetaClass(void) {
```
Where are other offsets? Please just add a case for your device + OS combination

benjamin-42 commented on Trident/offsetfinder.c:
```diff
@@ -1117,3 +1123,14 @@ uint32_t find_mac_proc_check(void) {
     }
 }
+
+    switch (target_environment) {
```
Should be part of the main switch for this function

EGYbkgo9449 replied:
I've wrote ur offsetfinder.c and changed only the wrong offsets -_-

i0mar committed:
Update offsetfinder.c

benjamin-42 commented:
I rebased your work to get only one commit and to remove the changes made at the top of the file (you removed the comment), changes are available in master.
